tools/csvstoskyui/csvstoskyui.py

```diff
@@ -0,0 +1,143 @@
+import csv
+import json
+import re
+import argparse
+
+"""
+The content of topologyrules.conf file includes a python dictionary with the following keys: 
+
+filename    - The name of the file to read the csv information from
```

Comment:
s/from example/for example/

Reply:
Not following you... text is saying "read from" so I do not think it sound be change to "read for". Please explain more.